src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -398,32 +398,7 @@ async Task<MsalAccessTokenCacheItem> ITokenCacheInternal.FindAccessTokenAsync(Au
                     OnAfterAccess(args);
                 }
 
-                // this is OBO flow. match the cache entry with assertion hash,
```

Comment:
Not actually deleted, but refactored a bit and moved into a separate method below.